src/main/java/org/apache/rocketmq/dashboard/service/client/MQAdminInstance.java

```diff
@@ -16,29 +16,23 @@
  */
 package org.apache.rocketmq.dashboard.service.client;
 
-import org.apache.commons.lang3.StringUtils;
-import org.apache.rocketmq.acl.common.AclClientRPCHook;
-import org.apache.rocketmq.acl.common.SessionCredentials;
-import org.apache.rocketmq.client.exception.MQClientException;
+import org.apache.commons.pool2.impl.GenericObjectPool;
 import org.apache.rocketmq.client.impl.MQClientAPIImpl;
 import org.apache.rocketmq.client.impl.factory.MQClientInstance;
-import org.apache.rocketmq.remoting.RPCHook;
 import org.apache.rocketmq.remoting.RemotingClient;
-import org.apache.rocketmq.tools.admin.DefaultMQAdminExt;
 import org.apache.rocketmq.tools.admin.DefaultMQAdminExtImpl;
 import org.apache.rocketmq.tools.admin.MQAdminExt;
 import org.joor.Reflect;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class MQAdminInstance {
-    private static final ThreadLocal<DefaultMQAdminExt> MQ_ADMIN_EXT_THREAD_LOCAL = new ThreadLocal<DefaultMQAdminExt>();
-    private static final ThreadLocal<Integer> INIT_COUNTER = new ThreadLocal<Integer>();
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MQAdminInstance.class);
+    private static MQAdminExt mqAdminExt;
```

Comment:
static means this is a singleton object, but multiple threads will assign a value to this object, there will be concurrency problems here.